examples/webgl_postprocessing.html

```diff
@@ -79,18 +79,21 @@
 
 				// postprocessing
 
-				composer = new EffectComposer( renderer );
-				composer.addPass( new RenderPass( scene, camera ) );
+				composer = new PP.EffectComposer( renderer );
+				composer.addPass( new PP.RenderPass( scene, camera ) );
 
-				const effect1 = new ShaderPass( DotScreenShader );
-				effect1.uniforms[ 'scale' ].value = 4;
+				const dotScreenMaterialShader = new THREE.ShaderMaterial( DotScreenShader );
+				const effect1 = new PP.ShaderPass( dotScreenMaterialShader, 'tDiffuse' );
+				dotScreenMaterialShader.uniforms[ 'scale' ].value = 4;
 				composer.addPass( effect1 );
 
-				const effect2 = new ShaderPass( RGBShiftShader );
-				effect2.uniforms[ 'amount' ].value = 0.0015;
+				const rgbShiftMaterialShader = new THREE.ShaderMaterial( RGBShiftShader );
+				const effect2 = new PP.ShaderPass( rgbShiftMaterialShader, 'tDiffuse' );
+				rgbShiftMaterialShader.uniforms[ 'amount' ].value = 0.0015;
 				composer.addPass( effect2 );
 
-				const effect3 = new ShaderPass( GammaCorrectionShader );
+				const gammaCorrectionMaterialShader = new THREE.ShaderMaterial( GammaCorrectionShader );
+				const effect3 = new PP.ShaderPass( gammaCorrectionMaterialShader, 'tDiffuse' );
```

Comment:
When using `postprocessing`, GammaCorrectionShader should be omitted. The package handles `renderer.outputColorSpace` automatically.

Reply:
My bad, will fix it. Thank you for the insight ! 🤝